PlayCover/Views/App Views/PlayAppView.swift

```diff
@@ -136,13 +139,15 @@ struct PlayAppView: View {
             .sheet(isPresented: $showDeleteGenshinAccount) {
                 DeleteGenshinAccountView()
             }
-            .alert("alert.app.delete", isPresented: $showClearCacheAlert) {
-                Button("button.Proceed", role: .destructive) {
-                    app.clearAllCache()
-                    showClearCacheToast.toggle()
-                }
-                Button("button.Cancel", role: .cancel) { }
-            }
+
+        /*
+         * Since SwiftUI‘s native alert does not support being dismissed programmatically,
+         * an async method will immediately hide the dialog after clicking the button,
+         * this leads to the execution state of the task not being observed by the user,
+         * resulting in the risk of misoperation,
+         * so a customized sheet is used as an alternative.
+         */
+            .sheet(isPresented: $showClearCacheAlert) { ClearCacheModalView(app: $app) }
```

Comment:
The uninstaller and the cache clearer should be consistent in alert and progress view. It would be best to change this to the uninstaller format (regular alert with progress vm showing uninstall progress), or changing the uninstaller alert to match this. The `ClearCacheModelView` could be generalised to support both of these processes.